_episodes/01-welcome.md

```diff
@@ -20,111 +20,113 @@ keypoints:
 
 It’s important to first assess the prior knowledge of the workshop
 participants because this will influence (to some extent) how you
 motivate the activities and how you communicate with the attendees.
-For instance, participants who have taken or helped with a workshop
-will have a much better concept of what goes on during a workshop than
-those who have never experienced one.  Furthermore, someone who live
-codes frequently and someone who has never live coded will experience
-the live coding activities differently.
 
 > ## Background
->
-> *Have you ever participated in a Software Carpentry or Data Carpentry Workshop?*
->
-> 1.  Yes, I have taken a workshop.  
-> 2.  Yes, I have been a workshop helper.  
-> 3.  Yes, I organized a workshop.  
-> 4.  No, but I am familiar with what is taught at a workshop.  
-> 5.  No, and I am not familiar with what is taught at a workshop.  
->
+> 
 > *Which of these most accurately describes your teaching experience?*
 >
 > 1.  I have been a graduate or undergraduate teaching assistant for a university/college course.  
 > 2.  I have not had any teaching experience in the past.  
 > 3.  I have taught a seminar, workshop, or other short or informal course.  
 > 4.  I have been the instructor-of-record for my own university/college course.  
 > 5.  I have taught at the K-12 level.  
->
-> *I know how to write multiple choice questions with diagnostic power.*
->
-> 1.  Strongly agree  
-> 2.  Agree  
-> 3.  Neutral  
-> 4.  Disagree  
-> 5.  Strongly disagree   
->
-> *If you teach, how often do you incorporate live coding in your classrooms? If you don't teach, how often have your instructors used live coding?*
->
-> 1.   Frequently
-> 2.   Occasionally
-> 3.   Rarely
-> 4.   Never
+> 
+> *Which of these questions assesses flaws in a student's mental model of a domain?*
+> 
+> 1. I'm not sure what a mental model is.
+> 2. "In Python, what is the expected output for the following statement: 1 + '2'"
+> 	(a) '12' 
+> 	(b) TypeError 
+> 	(c) ‘3’ 
+> 	(d) 3
+> 3. "Rate your experience with the R programming language." 
+> 	(a) never used it 
+> 	(b) beginner 
+> 	(c) intermediate 
+> 	(d) expert
+> 4. "What does the unix command 'cut' do?"
+> 	(a) Extracts sections from each line of input. 
+> 	(b) Sorts fields of a line 
+> 	(c) Searches the input file for lines containing a match to a pattern
+> 	(d) Removes a given input from a line
+> 
 {: .discussion}
 
-Now that we have a better idea of everyone's prior knowledge and
-familiarity with some of the SWC and DC teaching practices, we're
-ready to begin our training. Our goal is that by the end, you will
-have acquired some knew knowledge, confidence, and skills that you can
-use in your teaching practice in general and in teaching SWC and DC
-workshops specifically.
-
-Let's dive into the theory and practice behind designing multiple
+Now that we have a better idea of everyone's prior knowledge, 
+let's dive into the training.  Our first topic will be 
```

Comment:
--> Our first topic will be Cognitive Development and Mental Models

(or at least not just MCQs, even though these are an element of it)